scripts/win32.bat

```diff
@@ -107,8 +127,8 @@ For Each TopLevelDrive In GetTopLevelDrives()
 		Wscript.Echo "mountpoints: []"
 	Else
 		Wscript.Echo "mountpoints:"
-		For Each Mountpoint In TopLevelDrive.Item("Mountpoints")
-			Wscript.Echo "  - path: """ & Mountpoint & """"
+		For Each Mountpoint In TopLevelDrive.Item("Mountpoints").GetArray()
+			Wscript.Echo "	- path: """ & Mountpoint & """"
```

Comment:
The thing is that the unit tests test the parser, but don't run the actual scripts (since we would need some way to mock connected devices, or something like that).